EU4toV2/Source/Mappers/MinorityPops/MinorityPopMapper.cpp

```diff
@@ -26,22 +26,22 @@ void mappers::MinorityPopMapper::registerKeys()
 	registerRegex("[a-zA-Z0-9\\_.:]+", commonItems::ignoreItem);
 }
 
-bool mappers::MinorityPopMapper::matchMinorityPop(V2::Pop& pop) const
+bool mappers::MinorityPopMapper::matchMinorityPop(std::shared_ptr<V2::Pop> pop) const
```

Comment:
As before, the reference is generally preferred.

Might want to revisit the overall structure of the function as it is. From the signature alone it'd be hard to understand what it's doing, which is bad.